virtcontainers-pause/README.md

```diff
@@ -0,0 +1,16 @@
+# cc-runtime
+
+This directory contains the sources to create rpm spec files and debian source
+control files to create ``pause`` binary from github.com/containers/virtcontainers.
```

Comment:
control files to create the ``pause`` binary from the github.com/containers/virtcontainers repository.